yt_dlp/extractor/whyp.py

```diff
@@ -0,0 +1,45 @@
+from .common import InfoExtractor
+from ..utils import traverse_obj, float_or_none, str_or_none, url_or_none
+
+
+class WhypIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?whyp\.it/tracks/(?P<id>\d+)'
+    _TEST = {
+        'url': 'https://www.whyp.it/tracks/18337/home-page-example-track-b4kq7',
+        'md5': 'c1187b42ebf8605284e3dc92aeb33d16',
+        'info_dict': {
+            'url': 'https://cdn.whyp.it/50eb17cc-e9ff-4e18-b89b-dc9206a95cb1.mp3',
+            'id': '18337',
+            'title': 'Home Page Example Track',
+            'description': 'Just an example track for the home page!\n\nCredit: https://file-examples.com/index.php/sample-audio-files/sample-mp3-download/',
```

Comment:
use md5 for long descriptions or other long metadata strings
```suggestion
            'description': 'md5:bd758000fb93f3159339c852b5b9133c',
```